endpoints/openrtb2/sample-requests/invalid-whole/imp-empty-object.json

```diff
@@ -10,5 +10,5 @@
     }
   },
   "expectedReturnCode": 400,
-  "expectedErrorMessage": "Invalid request: request.imp[0] missing required field: \"id\"\n"
+  "expectedErrorMessage": "Invalid request: Key path not found"
```

Comment:
Why is this error message changing? The new message hides the reason the request failed. Can we keep this the way it was?

Reply:
The team discussed this offline and decided that it would be best if we check if the error is a `KeyPathNotFoundError`, in which case we would generate an error with the message `Invalid request: request.imp[0] missing required field: \"id\"\n"`.